apollo-router/Cargo.toml

```diff
@@ -99,7 +99,7 @@ humantime = "2.1.0"
 humantime-serde = "1.1.1"
 hyper = { version = "0.14.27", features = ["server", "client"] }
 hyper-rustls = { version = "0.23.2", features = ["http1", "http2"] }
-indexmap = { version = "1.9.3", features = ["serde-1"] }
+indexmap = { version = "2.0.0", features = ["serde"] }
```

Comment:
Can someone else give me a signal that this swap makes sense?  It's required to update to 2.x; `serde-1` is gone and only `serde` exists now.  Tests aren't failing!

Reply:
RELEASE NOTE ABOUT REMOVAL OF serde-1
https://github.com/bluss/indexmap/blob/0604c5b0afc3cab7f45db8bf0391f068e20307f3/RELEASES.md?plain=1#L8
Also:
Original documentation of purpose of that flag:
https://github.com/bluss/indexmap/blob/0604c5b0afc3cab7f45db8bf0391f068e20307f3/RELEASES.md?plain=1#L354C9-L354C9

SUMMARY: replacing `serde-1` with `serde` is correct.

Reply:
thanks.  That was my rough understanding, too, just wanted to make sure my `serde` knowledge wasn't missing something obvious.